lib/bundler/cli/outdated.rb

```diff
@@ -59,18 +57,15 @@ def run
         definition_resolution.call
       end
 
-      Bundler.ui.info ""
+      Bundler.ui.info("")
 
       # Loop through the current specs
       gemfile_specs, dependency_specs = current_specs.partition do |spec|
-        current_dependencies.key? spec.name
+        current_dependencies.key?(spec.name)
       end
 
-      specs = if options["only-explicit"]
-        gemfile_specs
-      else
-        gemfile_specs + dependency_specs
-      end
+      specs = gemfile_specs
+      specs += dependency_specs unless options["only-explicit"]
```

Comment:
I think this change is only stylistic, and does not add much to readability. Can you remove it?